docs/mega-linter-runner.md

```diff
@@ -70,14 +75,14 @@ See [`.pre-commit-hooks.yaml`](../.pre-commit-hooks.yaml) for more details.
 mega-linter-runner [OPTIONS]
 ```
 
-The options are only related to mega-linter-runner. For MegaLinter options, please use a `.mega-linter.yml` [configuration file](#configuration)
+The options are only related to mega-linter-runner. For MegaLinter options, please use a `.mega-linter.yml` [configuration file](#configuration). If settings are defined in both option flags and .mega-linter.yml, the priority is on option flags. 
 
```

Comment:
I thought this was useful to clarify

Reply:
Yeah, that seems good to reinforce. Here is my proposed rewording from passive to active voice for the newly added sentence: "Option flags take precedence over the configuration file."

Reply:
Done